doc/latex.rst

```diff
@@ -1429,24 +1436,20 @@ own docs at https://www.sphinx-doc.org.
 .. _ellipse: https://ctan.org/pkg/ellipse
 
 
-The following legacy behavior is currently not customizable:
+The following legacy behavior applies:
 
-- For :rst:dir:`code-block`, padding and border-width and shadow (if one adds
-  one) will go into the margin; the code lines remain at the same place
-  independently of the values of the padding and border-width, except for
-  being shifted vertically of course to not overwrite other text due to the
-  width of the border or external shadow.
+- For :rst:dir:`code-block` or :rst:dir:`literalinclude`, padding and
+  border-width and shadow (if any) will go into the margin; the code
+  lines remain at the same place independently of the values of the padding
+  and border-width, except for being shifted vertically of course to not
+  overwrite other text due to the width of the border or external shadow.
 
-- For :dudir:`topic` (and contents_) the shadow (if on right) goes into the
-  page margin, but the border and the extra padding are kept within the text
-  area.  Same for admonitions.
+- For the other directives, the
+  shadow (if on right) goes into the page margin, but the border and the extra
+  padding are kept within the text area.
```

Comment:
```suggestion
- For the other directives, the shadow (if on right) goes into the page margin,
  but the border and the extra padding are kept within the text area.
```

The line split was a bit weird.

Reply:
note to myself "if on right" is weird, left shadow also is in margin, should be "horizontal shadow", to be checked